### PR TITLE
Fix incompatibilities with PHP 8.1

### PR DESCRIPTION
**Pull request by sjinks**
## Description

This PR fixes incompatibilities with PHP 8.1 described in #2667.

### Plugin Updated: VIP Init

Fix incompatibilities with PHP 8.1

### Plugin Updated: VIP Search

Fix incompatibilities with PHP 8.1

### Plugin Updated: VIP Helpers

Fix incompatibilities with PHP 8.1

## Checklist

Please make sure the items below have been covered before requesting a review:

- [x] This change works and has been tested locally (or has an appropriate fallback).
- [ ] This change works and has been tested on a Go sandbox.
- [ ] This change has relevant unit tests (if applicable).
- [ ] This change has relevant documentation additions / updates (if applicable).
- [x] I've created a changelog description that aligns with the provided examples.

## Steps to Test

The CI should pass; PHP 8.1 will still fail.

### CODE REVIEW

File: 000-debug/debug-mode.php [hunk 1]
```diff
@@ -25,8 +25,8 @@
 add_action( 'muplugins_loaded', __NAMESPACE__ . '\init_debug_mode' );
 
 function init_debug_mode() {
-	// phpcs:ignore WordPress.Security.NonceVerification.Recommended
-	$enable = filter_var( $_GET['a8c-debug'] ?? '', FILTER_SANITIZE_STRING );
+	// phpcs:ignore WordPress.Security.NonceVerification.Recommended, WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
+	$enable = $_GET['a8c-debug'] ?? '';
```
Comment (GaryJones) [inline 000-debug/debug-mode.php:29]: Could `sanitize_text_field()` be used here?

Comment (sjinks) [reply 000-debug/debug-mode.php:29]: @GaryJones yes, but we use `$enable` only to compare its value against two predefined constants, that is why I decided not to waste CPU cycles.

Comment (GaryJones) [reply 000-debug/debug-mode.php:29]: Makes sense.

**Review by pschoffer — APPROVED** [review]

**Comment by sjinks** [comment]
r2055-stacks

**Pull request merged**